forge-gui/res/cardsfolder/upcoming/pass_the_torch.txt

```diff
@@ -0,0 +1,12 @@
+Name:Pass the Torch
+ManaCost:1 R
+Types:Instant
+A:SP$ DealDamage | ValidTgts$ Creature,Player,Planeswalker | TgtPrompt$ Select any target | NumDmg$ 2 | SubAbility$ DBChoose | SpellDescription$ CARDNAME deals 2 damage to any target.
+SVar:DBChoose:DB$ ChooseCard | ChoiceZone$ Hand | Choices$ Creature.YouOwn | Mandatory$ True | ChoiceTitle$ Choose a creature card in your hand | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ FreeCastGrave | Name$ Pass the Torch's Perpetual Effect | Duration$ Permanent
```

Comment:
+ `SpellDescription$ It perpetually gains "Whenever this creature deals combat damage to a player, you may cast target card named Pass the Torch from your graveyard without paying its mana cost. If you do, this creature perpetually loses this ability".`

Reply:
Once you've added those SpellDescriptions, be sure to look at the card detail pane in Forge to ensure that the card text is correct